.github/workflows/ci-gh1.yml

```diff
@@ -0,0 +1,63 @@
+name: Build legate.core on GH
+
+concurrency:
+  group: ci-gpu-on-${{ github.event_name }}-from-${{ github.ref_name }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches-ignore:
+      - gh-pages  # deployment target branch (this workflow should not exist on that branch anyway)
+  # NOTE: this workflow will not work for pull_requests since GH pull requests do not have access to secrets.
+
+jobs:
+  build:
+    permissions: read-all
+    # Ref: https://docs.rapids.ai/resources/github-actions/#cpu-labels for `linux-amd64-cpu4`
+    runs-on: ${{ github.repository == 'nv-legate/legate.core' && 'linux-amd64-cpu4' || 'ubuntu-latest' }}
+    container:
+      options: -u root
+      image: pauletaylor/devcontainers:23.06-cpp-cuda11.8-mambaforge-ubuntu22.04
```

Comment:
Are you using pauletaylor rather than rapidsai for a specific reason?

Reply:
Yes, I switched the repo to verify a change. I'll be switching it back here shortly.

Reply:
Switched back to the rapidsai container.